src/sentry/migrations/0425_alter_user_id_bigint.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.28 on 2023-04-20 19:04
+
+from django.db import migrations
+
+import sentry.db.models.fields.bounded
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0424_drop_group_forecast_table"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="user",
+            name="id",
+            field=sentry.db.models.fields.bounded.BoundedBigAutoField(
+                primary_key=True, serialize=False
+            ),
+        ),
+    ]
```

Comment:
Changing the type on a primary key will lock the table while postgres rewrites all the data in the table. This could take a bit of time and we'd have uptime issues.

Looking at our schema in production it is already a bigint, but other environments would be wrong. Perhaps we can make this a `SeparateDatabaseAndState` migration that only modifies schema if the current column type is incorrect?